db/db_impl/db_impl.cc

```diff
@@ -2140,6 +2148,7 @@ Status DBImpl::CreateColumnFamilyImpl(const ColumnFamilyOptions& cf_options,
       ROCKS_LOG_INFO(immutable_db_options_.info_log,
                      "Created column family [%s] (ID %u)",
                      column_family_name.c_str(), (unsigned)cfd->GetID());
+      single_column_family_mode_.store(false, std::memory_order_release);
```

Comment:
why move this line from 2135?

Reply:
I forget why I place it here.... Maybe I can try to place it back to origin